core/core/src/ansible/roles/master/tasks/registry-secrets.yml

```diff
@@ -0,0 +1,7 @@
+- name: Apply docker-registry secrets
+  shell: kubectl --kubeconfig=/home/{{ admin_user.name }}/.kube/config create secret docker-registry {{ item.name }} --docker-server={{ item.server_url }} --docker-username={{ item.username }} --docker-password="{{ item.password }}" --docker-email={{ item.email }} --namespace={{ item.namespace }} --dry-run -o yaml | kubectl apply -f -
+  become_user: "{{ admin_user.name }}"
+  loop: "{{ kubernetes.image_registry_secrets }}"
```

Comment:
Same as for deployments, workaround is:
`loop: "{{ kubernetes.image_registry_secrets if kubernetes.image_registry_secrets else [] }}"`

Reply:
Will not be changed.